### PR TITLE
Fix breaking changes from Lisk SDK v6.0.0-alpha.32139 - Closes #659

### LINKED ISSUE #659: Fix breaking changes from Lisk SDK v6.0.0-alpha.32139
### Description

Update breaking changes introduced in Lisk SDK version `6.0.0-alpha.32139` to get Lisk Core working.

### Acceptance Criteria

Updating Lisk Core with the canary build (`6.0.0-alpha.32139`) fails since the SDK introduces several breaking changes

### Steps to reproduce

Update SDK to latest alpha version

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?

This PR resolves #659 

### How was it solved?

- [x] Updating Lisk Core to use SDK canary 6.0.0-alpha.32139
- [x] Refactor/fix all breaking changes
- [x] Update genesis block and config.json
- [x] Update unit tests

### How was it tested?
- npm install --registry https://npm.lisk.com/
- npm run build
- ./bin/run start -n devnet

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
Looks fine, please remove the unused code and ID related changes

File: config/alphanet/config.json [hunk 1]
```diff
@@ -17,7 +17,6 @@
 		"communityIdentifier": "Lisk",
 		"maxTransactionsSize": 15360,
 		"minFeePerByte": 1000,
-		"baseFees": [],
 		"modules": {}
```
Comment (sameersubudhi) [inline config/alphanet/config.json:20]: Can we please update the config to include the reward milestones?

File: src/application/modules/legacy/module.ts [hunk 2]
```diff
@@ -41,25 +39,29 @@ import {
...
+import { LegacyAccountStore } from './stores/legacyAccountStore';
 import { ReclaimCommand } from './commands/reclaim';
 import { RegisterKeysCommand } from './commands/register_keys';
 
 // eslint-disable-next-line prefer-destructuring
 const validator: liskValidator.LiskValidator = liskValidator.validator;
 
 export class LegacyModule extends BaseModule {
-	public name = MODULE_NAME_LEGACY;
 	public id = MODULE_ID_LEGACY_BUFFER;
```
Comment (sameersubudhi) [inline src/application/modules/legacy/module.ts:50]: Should we also not remove `id` here and its reference inside the `metadata`?

File: src/application/modules/legacy/module.ts [hunk 3]
```diff
@@ -112,7 +113,7 @@ export class LegacyModule extends BaseModule {
 		);
 
 		validator.validate(genesisLegacyStoreSchema, { legacySubstore });
-
+		const store = this.stores.get(LegacyAccountStore);
```
Comment (sameersubudhi) [inline src/application/modules/legacy/module.ts:116]: Can we please rename it to `legacyStore` instead and move the declaration to the bottom before it is used?

File: src/application/modules/legacy/module.ts (hunk 2 above)
Comment (nagdahimanshu) [reply src/application/modules/legacy/module.ts]: Removed all references of `id`

File: config/alphanet/config.json [hunk 4]
```diff
@@ -17,8 +17,14 @@
...
 		"maxTransactionsSize": 15360,
 		"minFeePerByte": 1000,
-		"baseFees": [],
-		"modules": {}
+		"modules": {
+			"reward": {
+				"tokenID": "0000000000000000",
+				"offset": 10,
+				"distance": 3000000,
+				"brackets": ["1000000000000", "400000000", "300000000", "200000000", "100000000"]
```
Comment (sameersubudhi) [inline config/alphanet/config.json:25]:
```suggestion
				"brackets": ["500000000", "400000000", "300000000", "200000000", "100000000"]
```

File: config/devnet/config.json [hunk 5]
```diff
@@ -17,8 +17,14 @@
...
 		"maxTransactionsSize": 15360,
 		"minFeePerByte": 1000,
-		"baseFees": [],
-		"modules": {}
+		"modules": {
+			"reward": {
+				"tokenID": "0000000000000000",
+				"offset": 10,
+				"distance": 3000000,
+				"brackets": ["1000000000000", "400000000", "300000000", "200000000", "100000000"]
```
Comment (sameersubudhi) [inline config/devnet/config.json:25]:
```suggestion
				"brackets": ["500000000", "400000000", "300000000", "200000000", "100000000"]
```

File: config/alphanet/config.json [hunk 6]
```diff
@@ -17,8 +17,14 @@
 		"communityIdentifier": "Lisk",
 		"maxTransactionsSize": 15360,
 		"minFeePerByte": 1000,
-		"baseFees": [],
-		"modules": {}
+		"modules": {
+			"reward": {
+				"tokenID": "0000000000000000",
+				"offset": 10,
```
Comment (sameersubudhi) [inline config/alphanet/config.json:23]: Let us keep the `offset` similar to the current version. But let it remain `10` for `devnet`.
```suggestion
				"offset": 2160,
```

**Review by sameersubudhi — CHANGES_REQUESTED** [review]

**Review by sameersubudhi — APPROVED** [review]
Thank you for the changes 🙏

**Review by ManuGowda — APPROVED** [review]

**Pull request merged**